bower.json

```diff
@@ -1,14 +1,22 @@
 {
   "name": "ember-ajax",
   "dependencies": {
+    "chai": "~2.3.0",
     "ember": "~2.6.0",
     "ember-cli-shims": "0.1.0",
+    "ember-mocha-adapter": "~0.3.1",
     "ember-qunit-notifications": "0.1.0",
-    "mocha": "~2.2.4",
-    "chai": "~2.3.0",
-    "ember-mocha-adapter": "~0.3.1"
+    "mocha": "~2.2.4"
   },
   "resolutions": {
     "ember": "~2.5.0"
-  }
+  },
+  "version": "2.5.2",
```

Comment:
I'd shy away from it personally because, knowing myself, I'd remember to update it in one file but forget to do the other. Depends on how goldfish-like everyone's memory is.